ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpStageSpec.scala

```diff
@@ -97,4 +99,47 @@ trait CommonFtpSourceSpec extends BaseSpec {
     }
   }
 
+  "FtpIOSink" should {
```

Comment:
@juanjovazquez @eharik I tried several approaches, chmod being one of them. But ultimately, the underlying filesystem is Jimfs - which [does not support posix-like permissions](https://github.com/google/jimfs).

> For example, while setting and reading POSIX file permissions is possible with the "posix" view, those permissions will not actually affect the behavior of the file system.

I don't see an easy way of testing this without moving to - say - [Memoryfilesystem](https://github.com/marschall/memoryfilesystem).
The good news is that - somehow obviously - when trying it with my local filesystem, I get IOExceptions for permission errors.

WDYT?